docs/SRS-Volere/SRS.tex

```diff
@@ -190,15 +190,27 @@ \subsection{Adaptability Requirements}
 
 \section{Security Requirements}
 \subsection{Access Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{ACR1}: The application shall require acceptance of terms and services to allow access to application's database at all times. 
+\end{itemize}
 \subsection{Integrity Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{INR1}: The application shall ensure data integrity by using encryption mechanisms for preventing unauthorized modifications.
+    \item \textbf{INR2}: The application shall be protected from actors with harmful intentions.
+\end{itemize}
 \subsection{Privacy Requirements}
-\lips
+\begin{itemize}
+    \item \textbf{PRR1}: The application shall maintain  proper data anonymization and sanitation techniques.
```

Comment:
extra space after proper ... not sure if this reflects in pdf tho ... 

approving anyways.